src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/handlers/presubmit/AppealSkeletonBundleGenerator.java

```diff
@@ -57,8 +61,9 @@ public boolean canHandle(
         requireNonNull(callbackStage, "callbackStage must not be null");
         requireNonNull(callback, "callback must not be null");
 
+        Event validEvent = isSaveAndContinueEnabled ? Event.SUBMIT_CASE : Event.BUILD_CASE;
```

Comment:
hmm, just to clarify do we want to keep logic same with default `isSaveAndContinueEnabled` as false.
If isSaveAndContinueEnabled is false by default then we should have it:
`isSaveAndContinueEnabled ? Event.BUILD_CASE : Event.SUBMIT_CASE` 
please clarify maybe I am wrong

Reply:
Currently, this logic runs for the SubmitCase event only. With this ticket, we put this logic behind the "isSaveAndContinueEnabled" feature flag. Therefore, when the feature value is true, it runs the current logic. However, when the value is false, this logic runs for the BuildCase event. 

For the release task, as agreed, I remove the feature flag from this service. It's not needed here because it is driven by the case-API service.